projects/wmf_default.yaml

```diff
@@ -126,9 +126,11 @@ paths:
                         Gumagamit:Lsjbot/Kartrutor2: true
                       commons.wikimedia.org:
                         Commons:Quality_images_candidates/candidate_list: true
+                        Commons:Quality_images/Subject/Places/Natural_structures: true
                         User:Darkweasel94/Vienna/2016_December_14-16: true
                         User:J_budissin/Uploads/BiH/2016_December_11-20: true
                         User:Jarekt/d: true
+                        User:Nicolas_Rück_(WMDE)/Supported_by_Wikimedia_Deutschland_2015/1: true
```

Comment:
I pasted User:Nicolas_R%C3%BCck_(WMDE)/Supported_by_Wikimedia_Deutschland_2015/2 in IRC .. but maybe both /1 and /2 are a problem.